source/cookbook/user_interface_and_interaction/resetting_scroll_on_route_changes.md

```diff
@@ -17,8 +17,10 @@ App.ResetScroll = Ember.Mixin.create({
 
 Only if you need do something on the `activate` method you must call `this._super()` at the beginning:
 
-```js
-App.IndexRoute = Ember.Route.extend(App.ResetScroll, {
+```app/routes/index.js
+import ResetScroll from 'app/mixins/reset-scroll.js';
```

Comment:
Import path should be relative (and not include the extension): `../mixins/reset-scroll'`